### ISSUE TITLE
Beeper switch

### ISSUE THREAD

**Issue by WildeRNS** [syssi/esphome-pipsolar#148]
Is any possibility to make beeper switch?

**Comment by syssi** [comment]
Do you know the commands to turn the beeper on/off? Is there current state of the beeper available via QPIGS or any other "retrieve the current state/configuration" command?

**Comment by WildeRNS** [comment]
No, but in project: https://github.com/softwarecrash/Solar2MQTT this function is realized.

**Comment by syssi** [comment]
What's the feature called at Solar2MQTT?

**Comment by WildeRNS** [comment]
This must be a custom mqqt command, I didn't try that project (I'm using your and happy with them), I saw beeper switch in that (https://youtu.be/9-4SgkW3FZo) video, and asked maybe it will be possible to add same?

**Comment by syssi** [comment]
Sorry! Without the knowledge about the required command this cannot be added.
